custom_components/bambu_lab/fan.py

```diff
@@ -32,17 +32,17 @@ class BambuLabFanEntityDescription(FanEntityDescription, BambuLabFanEntityDescri
 FANS: tuple[FanEntityDescription, ...] = (
     BambuLabFanEntityDescription(
         key="cooling_fan",
-        name="Cooling Fan",
-        value_fn=lambda device: device.fans.cooling_fan_speed
+        translation_key="cooling_fan",
+        value_fn=lambda device: device.fans.cooling_fan_speed,
```

Comment:
I think, but am not sure, that this extra comma will cause problems? But maybe not if this is running fine for you as-is. But it is inconsistent with the other code in this file.